styles/globals.scss

```diff
@@ -150,8 +152,12 @@ $xxl: 1400px;
     @apply leading-c3;
     @apply font-450 tracking-copy;
   }
-  
+
+  .full-width {
+    @apply ml-[50%] w-screen -translate-x-1/2;
+  }
+
   .full-width-bg {
-    @apply top-0 bottom-0 absolute left-1/2 -translate-x-1/2 w-[100vw] -z-10;
+    @apply absolute top-0 bottom-0 left-1/2 -z-10 w-[100vw] -translate-x-1/2;
   }
```

Comment:
could we consolidate these classes? i think they're the same except for the absolute positioning. maybe we could move `absolute top-0 bottom-0` to the footer?